code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -650,3 +650,23 @@
 	var/location = get_turf(holder.my_atom)
 	for(var/i in 1 to 10)
 		new /obj/item/stack/sheet/plastic(location)
+
+//Botany chemicals, ported from /VG/ #15432
+
+/datum/chemical_reaction/eznutrmient
+	name = "E-Z-Nutriment"
+	id = "eznutriment"
+	results = list("eznutriment" = 3)
+	required_reagents = list("nitrogen" = 1, "phosphorus" = 1, "potassium" = 1)
```

Comment:
The biogenerator can already make all plant nutriments if their dispenser runs out, doesn't sound necessary.